### PR TITLE
Added missing social link in hamburger :D

### PR DESCRIPTION
**Pull request by yashk0thari**
#98 - I've solved/contributed to this issue :D

### CODE REVIEW

**Comment by yashk0thari** [comment]
https://beta-king.github.io/pattfd2014.fossasia.org/ --> Preview Link :D

**Review by realslimshanky — APPROVED** [review]
LGTM

**Review by sayak119 — APPROVED** [review]

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@Beta-King  Please maintain alignment and position/gap of icons as they are looking very close now.
image

**Comment by yashk0thari** [comment]
they are close vertically right ?? Changing that right now :D

**Comment by yashk0thari** [comment]
https://beta-king.github.io/pattered.2014.fossasia.org/ --> Made changes; now they are aligned :D

**Comment by yashk0thari** [comment]
Squashed too :D

**Comment by yashk0thari** [comment]
Please review and merge :D

File: _includes/header.html [hunk 1]
```diff
@@ -343,8 +343,9 @@
                     <li><a href="http://facebook.com/fossasia" target="_self"><i class="icon social_facebook"></i></a></li>
                     <li><a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_self"><i class="icon social_youtube icon-large"></i></a></li>
                     <li><a href="https://www.flickr.com/photos/fossasia/" target="_self"><i class="icon social_flickr"></i></a></li>
-                    <li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li>
+                    <li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li><br><br>
```
Comment (vaibhavsingh97) [inline _includes/header.html:346]: don't use `<br>` here. Also, add space between icons.
Refrence:
image

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 1 above)
Comment (yashk0thari) [reply _includes/header.html:346]: Then how should i align them ??

**Comment by yashk0thari** [comment]
I dont currently know how to that; ill learn it and make changes, till then im closing this PR :D

**Pull request closed without merging**